java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -51,4 +56,32 @@ public void custom_command_ping() {
         var data = clusterClient.customCommand(new String[] {"ping"}).get(10, TimeUnit.SECONDS);
         assertEquals("PONG", data.getSingleValue());
     }
+
+    @Test
+    @SneakyThrows
+    public void ping() {
```

Comment:
To avoid test duplication for cluster and standalone should we create a BaseClientTest with parametrized tests in this PR or should we refactor it later?

Reply:
I'll do that now.  We have time.